assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -110,37 +98,19 @@ export default function FormInstructions( { showExistingTagMessage } ) {
 		return (
 			<p>
 				{ __(
-					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below. You can change these later in your settings.',
+					'Looks like your site is using paired AMP. Please select your Tag Manager account and relevant containers below.',
 					'google-site-kit'
 				) }
 			</p>
 		);
 	}
 
 	return (
-		<Fragment>
-			{ showExistingTagMessage && hasExistingTag && (
-				<p>
-					{ sprintf(
-						// translators: %s: the existing container ID.
-						__(
-							'An existing tag was found on your site (%s). If you later decide to replace this tag, Site Kit can place the new tag for you. Make sure you remove the old tag first.',
-							'google-site-kit'
-						),
-						containerID
-					) }
-				</p>
+		<p>
+			{ __(
+				'Please select your Tag Manager account and container below.',
```

Comment:
Same here. Shouldn't it be `Please select your Tag Manager account and container below. You can change these later in your settings.`?

Reply:
See above comment

Reply:
Yep, works for me.